contributing/contributing.md

```diff
@@ -1,3 +1,9 @@
+---
```

Comment:
Ah.. this file is a required file for all Google open source github repos. I believe it is meant to be specific to the docsite repo, whereas we want a "Contributing" section that actually mostly deal with our data repo.

Maybe we could differentiate by renaming the current file to CONTRIBUTING.md and adding a new one contributing_to_datacommons.md ?

Reply:
Or we could even just start with your contributing_datasets.md file!

Reply:
Hi, I retained the contributing.md in the main directory as CONTRIBUTING.md and also copied that content into a file following your suggested title.

Reply:
Could we keep this as contributing.md, and instead rename the contributing_datasets.md to adding_datasets.md (or another inviting verb, importing_datasets.md)?

Reply:
This is a great idea! I went ahead and renamed the file.